test/bats/tests/akeyless/pod-inline-volume-secretproviderclass.yaml

```diff
@@ -0,0 +1,29 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: secrets-store-inline
+spec:
+  containers:
+  - image: k8s.gcr.io/e2e-test-images/busybox:1.29
+    name: busybox
+    imagePullPolicy: IfNotPresent
+    command:
+    - "/bin/sleep"
+    - "10000"
+    volumeMounts:
+    - name: secrets-store-inline
+      mountPath: "/mnt/secrets-store"
+      readOnly: true
+    env:
+      - name: AKEYLESS_ACCESS_KEY
+        valueFrom:
+          secretKeyRef:
+            name: credentials
+            key: akeyless-test-cred
```

Comment:
This credential is not required by the workload pod. The CSI provider is the one that users the credentials to get the secret. 

I've opened this PR to add an optional job for akeyless provider: https://github.com/kubernetes/test-infra/pull/25751. The required credentials will be injected into the test pod and value should be available in env var `AKEYLESS_ACCESS_KEY`. You can use that value to create a Kubernetes secret and reference that Kubernetes with `nodePublishSecretRef`.

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/3d93b4a03a68fd0de22fc09711734db0d2f51f70/test/bats/azure.bats#L62-L69

That secret would be referenced in the volume mount like this:
https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/3d93b4a03a68fd0de22fc09711734db0d2f51f70/test/bats/tests/azure/pod-secrets-store-inline-volume-crd.yaml#L25-L26

The driver will pass the value of that secret in https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/3d93b4a03a68fd0de22fc09711734db0d2f51f70/provider/v1alpha1/service.proto#L48-L49

Reply:
removed

Reply:
added support for k8s secret